source/modes/GameEditorModeBase.cpp

```diff
@@ -132,6 +133,12 @@ GameEditorModeBase::~GameEditorModeBase()
     // Delete the potential pending event messages
     for (EventMessage* message : mEventMessages)
         delete message;
+
+    // Clear up any events and chat messages.
+    mRootWindow->getChild("GameChatWindow/GameChatText")->setText("");
```

Comment:
Please pay attention to not upload code that have been changed. You always complain that you are too much reviewed but I can't remember any PR without loads of old code restored. I don't know how you setup your dev environment but there is clearly a problem.